fhir-persistence-schema/README.md

```diff
@@ -223,5 +223,11 @@ Note: the jar file is stored locally in `fhir-persistence-schema/target` or in t
     - grants.sql:  `db2 -tvf grants.sql`
     - stored-procedures.sql:  `db2 -td@ -vf stored-procedures.sql`
 
+### Extended Properties
+You may pass additional properties via the commandline or in the properties file: 
+
+| Property | Description |
+|---------|---------|
+| ssl | Y or anything else, Y triggers JDBC sslConnection=true, an example --prop ssl=Y| 
```

Comment:
Should we just add it to the table at in the `Connection properties` section?
Are there any other properties we support but don't document there?

Reply:
sure, let me move it around

Reply:
Fixed